tests/distributions/test_gaussian_mixtures.py

```diff
@@ -5,11 +5,11 @@
 import torch
 
 import pytest
-from pyro.distributions import MixtureOfDiagNormalsSharedCovariance
+from pyro.distributions import MixtureOfDiagNormalsSharedCovariance, GaussianScaleMixture
 from tests.common import assert_equal
 
 
-@pytest.mark.parametrize('mix_dist', [MixtureOfDiagNormalsSharedCovariance])
+@pytest.mark.parametrize('mix_dist', [MixtureOfDiagNormalsSharedCovariance, GaussianScaleMixture])
 @pytest.mark.parametrize('K', [3])
 @pytest.mark.parametrize('D', [2, 4])
```

Comment:
Can you add `D=1` example to test your `dim > 1` conditional logic?

Reply:
added D=1 assert error